reconcile/utils/terrascript_aws_client.py

```diff
@@ -5687,22 +5698,39 @@ def populate_tf_resource_rosa_authenticator(self, spec):
         )
         tf_resources.append(api_gateway_integration_token_response_resource)
 
-        # AUTH
-        api_gateway_integration_auth_response_resource = aws_api_gateway_integration_response(
+        # AUTH - OCM
+        ocm_api_gw_intg_auth_response_resource = aws_api_gateway_integration_response(
             "gw_integration_response_auth",
             rest_api_id=f"${{{api_gateway_rest_api_resource.id}}}",
             resource_id=f"${{{api_gateway_auth_resource.id}}}",
             http_method="${aws_api_gateway_method.gw_method_auth_get.http_method}",
             status_code="${aws_api_gateway_method_response.gw_method_auth_get_response.status_code}",
             response_parameters={
                 "method.response.header.Location": f"'{user_pool_url}/oauth2/authorize?client_id="
-                f"${{{cognito_user_pool_client.id}}}\u0026response_type=code"
+                f"${{{ocm_cognito_user_pool_client.id}}}\u0026response_type=code"
+                f"\u0026scope=openid+gateway/AccessToken\u0026redirect_uri={bucket_url}/"
+                "token.html'",
+            },
+            depends_on=["aws_api_gateway_integration.gw_integration_auth"],
+        )
+        tf_resources.append(ocm_api_gateway_integration_auth_response_resource)
+
+        # AUTH - INSIGHTS
```

Comment:
We shouldn't need this here, as it will be something we'll setup in our account. Correct me if I'm wrong, @lphiri.

Reply:
Correct,  we will have our own instance of API Gateway.

Reply:
Oh, okay. I think I maybe misunderstood what you folks were looking for then. I thought you were looking for a new user-pool-client associated with the ocm-api-gateway.

But if you actually just need a new user-pool-client associated with your insights-api-gateway, would it probably be most appropriate to just define the terraform HCL for the new user-pool-client alongside the terraform HCL that's already defining your insights-api-gateway?